code/src/index.css

```diff
@@ -8,6 +8,5 @@ body {
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: 'Barlow Semi Condensed', sans-serif;
```

Comment:
You could have added font-color here as well (instead of putting in in the components) in order to shorten the code. :)